GNUmakefile

```diff
@@ -11,6 +11,10 @@ build: lint
 test: lint
 	go test $(TESTARGS) -timeout=30s $(TEST)
 
+# Used in CI to prevent lint failures from being interpreted as test failures
+testnolint:
```

Comment:
Can we call this step from `test`? That will avoid the two diverging.

Reply:
Done